tests/unit/amo/components/TestFeaturedCollectionCard.js

```diff
@@ -55,6 +55,20 @@ describe(__filename, () => {
     expect(root.find(LandingAddonsCard)).toHaveProp('className', className);
   });
 
+  it("renders a theme class name to LandingAddonsCard if the collection's isTheme property is true", () => {
```

Comment:
This is still a little inaccurate. You are passing a value for `isTheme` directly into the component, so it's not really accurate to say "if the collection's isTheme property is true" because the component doesn't know where this `isTheme` value comes from. I think a better description might be "renders a theme class name to LandingAddonsCard when isTheme is true".

Reply:
yikes..I will get this : ) Thanks for suggestion : ) I will update..